src/Shared/BuildEnvironmentHelper.cs

```diff
@@ -384,6 +388,7 @@ private static string GetEnvironmentVariable(string variable)
         private static Func<string> s_getAppContextBaseDirectory = GetAppContextBaseDirectory;
         private static Func<IEnumerable<VisualStudioInstance>> s_getVisualStudioInstances = VisualStudioLocationHelper.GetInstances;
         private static Func<string, string> s_getEnvironmentVariable = GetEnvironmentVariable;
+        private static bool s_runningTests = CheckIfRunningTests();
```

Comment:
Review note: production code change (and others in b8d01363d9781b38e202a692b07914924eb16df5)